pkg/k8s/apis/cilium.io/v1alpha1/types.go

```diff
@@ -206,6 +206,8 @@ type ArgSelector struct {
 type ActionSelector struct {
 	// +kubebuilder:validation:Enum=Post;FollowFD;UnfollowFD;Sigkill;CopyFD;Override;GetUrl;DnsLookup;NoPost;Signal;TrackSock;UntrackSock;NotifyEnforcer;CleanupEnforcerNotification
 	// Action to execute.
+	// NOTE: actions FollowFD, UnfollowFD, and CopyFD are marked as deprecated and planned to
```

Comment:
could you try `kubebuilder:deprecatedversion`?

```
// +kubebuilder:deprecatedversion
marks this version as deprecated.
```

It's a bit unclear to me if that's appropriate for a field (apparently OpenAPIv3 support deprecated fields) or the whole CRD object.

Reply:
ah sorry indeed I didn't think correctly on this one, I went straight into deprecating a fields, nevermind!